projects/demo/src/app/home/home.component.html

```diff
@@ -26,23 +28,69 @@ <h2>Easily adapt ngx-valdemort to your preferences</h2>
 
 <demo-configuration></demo-configuration>
 
-<h2>Features</h2>
-
-<ul class="features">
-  <li>Minimum markup needed</li>
-  <li>Consistent, default error messages</li>
-  <li>Generic or control-specific default error messages by specifying a label</li>
-  <li>Overriding of default messages for custom error messages</li>
-  <li>All error messages have access to the error value (min, max, required length, etc.)</li>
-  <li>I18n possible</li>
-  <li>HTML in messages possible</li>
-  <li>Use of pipes in messages possible</li>
-  <li>Usable in reactive forms, template-driven forms, or standalone controls</li>
-  <li>Usable on controls nested in form groups or form arrays</li>
-  <li>Central configuration service</li>
-  <li>Free, and open-source. <a href="https://github.com/Ninja-Squad/ngx-valdemort">The code is available on Github</a></li>
-</ul>
-
-<h2>You convinced me. How to get started?</h2>
+<div class="row mt-5">
+  <div class="col-md-4">
```

Comment:
Doesn't change much, but since you want just columns of equal widths, you can just use col-md.